### ISSUE TITLE
do you have some visual C compiler builtin or not ?

### ISSUE THREAD

**Issue by Sandy4321** [winpython/winpython#801]
can you help understand , do you have some visual C compiler built in or not
for example will it work for xgboost for computer without installed Microsoft Visual C++ Redistributable for Visual Studio 2017?

from this link 
https://www.lfd.uci.edu/~gohlke/pythonlibs/#xgboost
Xgboost, a distributed gradient boosting (GBDT, GBRT or GBM) library.
Requires the Microsoft Visual C++ Redistributable for Visual Studio 2017.
xgboost‑0.90‑cp38‑cp38‑win_amd64.whl
xgboost‑0.90‑cp38‑cp38‑win32.whl
xgboost‑0.90‑cp37‑cp37m‑win_amd64.whl

**Comment by stonebig** [comment]
Go and see, do your reality check.

For the title question, there is no C-compiler in current WinPython (3.7/3.8) itself

**Comment by Sandy4321** [comment]
in any case if one computer has C++ VS compiler and python packages is successfully installed on this computer 

then if we have winpython on another computer but without C++ VS compiler

which folder should be copied to another computer 
to make it works on computer without C++ VS compiler
for example in folder
WPy64-3741\python-3.7.4.amd64\Lib\site-packages
I have 
joblib\
joblib-0.13.2.dist-info\
kiwisolver-1.1.0.dist-info\
lightgbm\
lightgbm-2.3.1.dist-info\
locket\
locket-0.2.0.dist-info\
markupsafe\
MarkupSafe-1.1.1.dist-info\
matplotlib\
matplotlib-3.1.2.dist-info\
mpl_toolkits\

will it be enough to copy  

lightgbm\
lightgbm-2.3.1.dist-info\

to make package lightgbm works on another computer ?
